components/markdown/tests/snapshots/markdown__all_markdown_features_integration.snap

```diff
@@ -83,7 +81,7 @@ line 1 of code
 line 2 of code
 line 3 of code
 </code></pre>
-<p>Block code &quot;fences&quot;</p>
+<p>Block code "fences"</p>
```

Comment:
how come it's not escaping anymore here?

Reply:
I believe this is due to a behavior change in pulldown-cmark, rather than the changes I made in this pull request. I think it might be https://github.com/pulldown-cmark/pulldown-cmark/pull/830, specifically. It introduces a new `escape_html_body_text` which should be used for escaping text nodes and which only escapes '<', '>', and '&', as opposed to `escape_html` which should be used for escaping HTML attributes and which escapes single and double quotes as well.

I believe that pulldown-cmark's `push_html` will now convert Event::Text nodes to HTML using `escape_html_body_text` (see src/html.rs in that pull request), hence why we see these quotes not being escaped anymore. This is also why we need to handle the alt attribute's text manually now, to ensure it gets attribute-escaped rather than body-escaped (which is now the default).

And FWIW, all other existing uses of `escape_html` that I could find in Zola were for escaping HTML attributes, so I think none of them need to be switched to `escape_html_body_text`.